rasa/core/trackers.py

```diff
@@ -398,7 +401,9 @@ def generate_all_prior_trackers(
     def applied_events(self) -> List[Event]:
```

Comment:
@Ghostvv Could you have a short look please? I also added some tests to `test_trackers` where you can see how the `applied_events()` vs the raw tracker

Reply:
left comments